shared-libs/cht-datasource/src/person.ts

```diff
@@ -109,5 +92,50 @@ export namespace v1 {
    * @returns a function for retrieving a paged array of people
    * @throws Error if a data context is not provided
    */
-  export const getPage = getPeople(Local.Person.v1.getPage, Remote.Person.v1.getPage);
+  export const getPage = (
```

Comment:
As in the index, it would be nice to add some docs here encouraging folks to use the other function:

> @see {@link getByType} which provides the same data, but without having to manually account for paging